### PR TITLE
[Firewall] Add note about challenge actions

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-3445.

### CODE REVIEW

File: content/firewall/cf-firewall-rules/actions.md [hunk 1]
```diff
@@ -213,3 +216,12 @@ For reference information on rule actions available for Cloudflare products powe
...
 {{</table-wrap>}}
+
+## Notes about challenge actions
+
+When you configure a firewall rule with one of the challenge actions — _Managed Challenge_, _JS Challenge_, or _Legacy CAPTCHA_ — and a request matches the rule, one of two things can happen:
+
+* The request is blocked if the visitor fails the challenge
+* The request is allowed if the visitor passes the challenge
+
+In this last case, no further firewall rules will be processed. This means that the action of any later rules with a challenge or _Block_ action also matching the request will not be applied, and request will be allowed.
```
Comment (marciocloudflare) [inline content/firewall/cf-firewall-rules/actions.md]:
```suggestion
In this last case, no further firewall rules will be processed. This means that the action of any later rules with a challenge or _Block_ action also matching the request will not be applied, and the request will be allowed.
```

Comment (marciocloudflare) [reply content/firewall/cf-firewall-rules/actions.md]: Not sure of I'm reading this right, but feels like the `the` is missing there

**Review by marciocloudflare — APPROVED** [review]

File: content/firewall/cf-firewall-rules/actions.md (hunk 1 above)
Comment (pedrosousa) [reply content/firewall/cf-firewall-rules/actions.md]: It is! Thank you.

**Review by migueldemoura — APPROVED** [review]
Great to see this added. 😝

**Pull request merged**